src/ILCompiler/repro/Program.cs

```diff
@@ -4,10 +4,13 @@
 
 using System;
 
-internal class Program
+namespace repro
 {
-    private static void Main(string[] args)
+    internal class Program
```

Comment:
Could you please revert the change in this file? (The other change in hello.cs is good.)